### PR TITLE
initial import from private repo

### PR DESCRIPTION
**Pull request by philomates**

### CODE REVIEW

File: test/umschreiben_clj/requires_test.clj [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+         '(ns aa (:using a.b.c))      'x.y.z  '(ns aa (:using a.b.c)(:require x.y.z))))
+
+  (testing "if the namespace is already required behaves like `replace`"
+    (let [namespace (code->node '(ns aa (:require [a.b.c :as c])) '(def bb (c/myfn 2)))]
+      (is (match? '(do (ns aa (:require [a.b.c :refer [myfn]])) (def bb (myfn 2)))
+                  (n/sexpr (requires/add namespace '[a.b.c :refer [myfn]]))))
+      (is (= (n/sexpr (requires/replace namespace 'a.b.c '[a.b.c :refer [myfn]]))
+             (n/sexpr (requires/add namespace '[a.b.c :refer [myfn]]))))))
+
+  ;; I don't find a use case for this! unless we want to provide low-level functions that should be composed. But
```
Comment (gabrielgiussi) [inline test/umschreiben_clj/requires_test.clj]: This is from me, let's remove the comment and/or the test. Do you think the test makes sense?

**Review by gabrielgiussi — CHANGES_REQUESTED** [review]
Configure a github action to run the tests

**Review by gabrielgiussi — CHANGES_REQUESTED** [review]
Configure a github action to run the tests

**Comment by philomates** [comment]
@gabrielgiussi I've updated things, good to go now?

**Review by gabrielgiussi — APPROVED** [review]

**Pull request merged**